### ISSUE TITLE
Kernel Issue - Bad GSO

### ISSUE THREAD

**Issue by greggitter** [home-assistant/operating-system#3532]
### Describe the issue you are experiencing

After updating to HAOS 13.0, this error seems to have started presenting itself along with very slow restart time.  This wasn't obvious until the proxmox host was restarted.  From the log....

![image](https://github.com/user-attachments/assets/96e9b522-9806-4c10-a3e7-16592222c78a)

This bug was replied on the forums, including here as possible issue/cause:

https://bugzilla.kernel.org/show_bug.cgi?id=219129

### What operating system image do you use?

generic-x86-64 (Generic UEFI capable x86-64 systems)

### What version of Home Assistant Operating System is installed?

6.6.44-haos, Home Assistant OS 13.0

### Did the problem occur after upgrading the Operating System?

Yes

### Hardware details

Proxmox 8 (latest)

### Steps to reproduce the issue

1.  Reboot proxmox hardware, problem occurs...but messages continue (see above)
2.
3.
...

### Anything in the Supervisor logs that might be useful for us?

```txt
Nothing related.
```

### Anything in the Host logs that might be useful for us?

```txt
2024-08-14 23:52:39.374 homeassistant kernel: enp6s18: bad gso: type: 1, size: 19
2024-08-14 23:52:40.355 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:52:41.353 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:52:45.689 homeassistant kernel: enp6s18: bad gso: type: 1, size: 48
2024-08-14 23:52:46.354 homeassistant kernel: enp6s18: bad gso: type: 1, size: 141
2024-08-14 23:52:47.853 homeassistant kernel: enp6s18: bad gso: type: 1, size: 141
2024-08-14 23:52:49.353 homeassistant kernel: enp6s18: bad gso: type: 1, size: 141
2024-08-14 23:52:49.354 homeassistant kernel: enp6s18: bad gso: type: 1, size: 141
2024-08-14 23:52:50.853 homeassistant kernel: enp6s18: bad gso: type: 1, size: 141
2024-08-14 23:52:52.353 homeassistant kernel: enp6s18: bad gso: type: 1, size: 141
2024-08-14 23:52:55.379 homeassistant kernel: enp6s18: bad gso: type: 1, size: 61
2024-08-14 23:53:05.675 homeassistant kernel: enp6s18: bad gso: type: 1, size: 106
2024-08-14 23:53:06.355 homeassistant kernel: enp6s18: bad gso: type: 1, size: 133
2024-08-14 23:53:06.356 homeassistant kernel: enp6s18: bad gso: type: 1, size: 141
2024-08-14 23:53:07.354 homeassistant kernel: enp6s18: bad gso: type: 1, size: 133
2024-08-14 23:53:07.355 homeassistant kernel: enp6s18: bad gso: type: 1, size: 141
2024-08-14 23:53:07.373 homeassistant kernel: enp6s18: bad gso: type: 1, size: 52
2024-08-14 23:53:08.355 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:53:09.354 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:53:10.354 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:53:11.354 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:53:12.354 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:53:12.355 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:53:14.354 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:53:18.354 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:53:23.407 homeassistant kernel: enp6s18: bad gso: type: 1, size: 106
2024-08-14 23:53:24.357 homeassistant kernel: enp6s18: bad gso: type: 1, size: 133
2024-08-14 23:53:24.357 homeassistant kernel: enp6s18: bad gso: type: 1, size: 141
2024-08-14 23:53:24.367 homeassistant kernel: enp6s18: bad gso: type: 1, size: 58
2024-08-14 23:53:35.692 homeassistant kernel: enp6s18: bad gso: type: 1, size: 48
2024-08-14 23:53:53.717 homeassistant kernel: enp6s18: bad gso: type: 1, size: 42
2024-08-14 23:53:54.358 homeassistant kernel: enp6s18: bad gso: type: 1, size: 141
2024-08-14 23:53:55.356 homeassistant kernel: enp6s18: bad gso: type: 1, size: 141
2024-08-14 23:53:56.356 homeassistant kernel: enp6s18: bad gso: type: 1, size: 141
2024-08-14 23:53:56.357 homeassistant kernel: enp6s18: bad gso: type: 1, size: 141
2024-08-14 23:53:57.356 homeassistant kernel: enp6s18: bad gso: type: 1, size: 141
2024-08-14 23:53:58.356 homeassistant kernel: enp6s18: bad gso: type: 1, size: 141
2024-08-14 23:54:00.356 homeassistant kernel: enp6s18: bad gso: type: 1, size: 141
2024-08-14 23:54:04.356 homeassistant kernel: enp6s18: bad gso: type: 1, size: 141
2024-08-14 23:54:09.375 homeassistant kernel: enp6s18: bad gso: type: 1, size: 40
2024-08-14 23:54:10.358 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:54:11.356 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:54:12.356 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:54:12.357 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:54:13.356 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:54:14.357 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:54:16.357 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:54:20.357 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:54:35.706 homeassistant kernel: enp6s18: bad gso: type: 1, size: 24
2024-08-14 23:54:36.372 homeassistant kernel: enp6s18: bad gso: type: 1, size: 55
2024-08-14 23:54:37.359 homeassistant kernel: enp6s18: bad gso: type: 1, size: 133
2024-08-14 23:54:37.359 homeassistant kernel: enp6s18: bad gso: type: 1, size: 129
2024-08-14 23:54:37.359 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:54:38.357 homeassistant kernel: enp6s18: bad gso: type: 1, size: 129
2024-08-14 23:54:38.358 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:54:39.358 homeassistant kernel: enp6s18: bad gso: type: 1, size: 129
2024-08-14 23:54:39.359 homeassistant kernel: enp6s18: bad gso: type: 1, size: 129
2024-08-14 23:54:39.359 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:54:41.358 homeassistant kernel: net_ratelimit: 2 callbacks suppressed
2024-08-14 23:54:41.358 homeassistant kernel: enp6s18: bad gso: type: 1, size: 129
2024-08-14 23:54:41.358 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:54:43.358 homeassistant kernel: enp6s18: bad gso: type: 1, size: 129
2024-08-14 23:54:47.358 homeassistant kernel: enp6s18: bad gso: type: 1, size: 129
2024-08-14 23:54:47.358 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:54:51.366 homeassistant kernel: kauditd_printk_skb: 4 callbacks suppressed
2024-08-14 23:54:51.366 homeassistant kernel: audit: type=1334 audit(1723679691.364:294): prog-id=78 op=LOAD
2024-08-14 23:54:51.372 homeassistant systemd-timesyncd[1056]: Network configuration changed, trying to establish connection.
2024-08-14 23:54:51.373 homeassistant systemd[1]: Started Journal Gateway Service.
2024-08-14 23:54:51.403 homeassistant systemd-timesyncd[1056]: Contacted time server 172.20.100.1:123 (172.20.100.1).
2024-08-14 23:54:55.383 homeassistant kernel: enp6s18: bad gso: type: 1, size: 106
2024-08-14 23:54:56.360 homeassistant kernel: enp6s18: bad gso: type: 1, size: 120
2024-08-14 23:54:56.360 homeassistant kernel: enp6s18: bad gso: type: 1, size: 141
2024-08-14 23:54:57.358 homeassistant kernel: enp6s18: bad gso: type: 1, size: 141
2024-08-14 23:54:58.358 homeassistant kernel: enp6s18: bad gso: type: 1, size: 141
2024-08-14 23:54:59.358 homeassistant kernel: enp6s18: bad gso: type: 1, size: 141
2024-08-14 23:54:59.375 homeassistant kernel: enp6s18: bad gso: type: 1, size: 24
2024-08-14 23:55:00.360 homeassistant kernel: enp6s18: bad gso: type: 1, size: 129
2024-08-14 23:55:00.360 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:55:01.358 homeassistant kernel: enp6s18: bad gso: type: 1, size: 129
2024-08-14 23:55:01.359 homeassistant kernel: enp6s18: bad gso: type: 1, size: 129
2024-08-14 23:55:01.360 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:55:05.692 homeassistant kernel: enp6s18: bad gso: type: 1, size: 55
2024-08-14 23:55:06.360 homeassistant kernel: enp6s18: bad gso: type: 1, size: 141
2024-08-14 23:55:15.705 homeassistant kernel: enp6s18: bad gso: type: 1, size: 48
2024-08-14 23:55:16.382 homeassistant kernel: enp6s18: bad gso: type: 1, size: 57
2024-08-14 23:55:17.360 homeassistant kernel: enp6s18: bad gso: type: 1, size: 129
2024-08-14 23:55:18.859 homeassistant kernel: enp6s18: bad gso: type: 1, size: 129
2024-08-14 23:55:18.860 homeassistant kernel: enp6s18: bad gso: type: 1, size: 129
2024-08-14 23:55:18.860 homeassistant kernel: enp6s18: bad gso: type: 1, size: 137
2024-08-14 23:55:25.698 homeassistant kernel: enp6s18: bad gso: type: 1, size: 103
2024-08-14 23:55:35.743 homeassistant kernel: enp6s18: bad gso: type: 1, size: 19
2024-08-14 23:55:36.362 homeassistant kernel: enp6s18: bad gso: type: 1, size: 121
2024-08-14 23:55:37.860 homeassistant kernel: enp6s18: bad gso: type: 1, size: 121
2024-08-14 23:55:37.861 homeassistant kernel: enp6s18: bad gso: type: 1, size: 140
2024-08-14 23:55:37.861 homeassistant kernel: enp6s18: bad gso: type: 1, size: 121
2024-08-14 23:55:37.861 homeassistant kernel: enp6s18: bad gso: type: 1, size: 140
2024-08-14 23:55:39.360 homeassistant kernel: enp6s18: bad gso: type: 1, size: 121
2024-08-14 23:55:40.860 homeassistant kernel: enp6s18: bad gso: type: 1, size: 121
2024-08-14 23:55:40.860 homeassistant kernel: enp6s18: bad gso: type: 1, size: 140
2024-08-14 23:55:42.360 homeassistant kernel: enp6s18: bad gso: type: 1, size: 121
2024-08-14 23:55:45.360 homeassistant kernel: enp6s18: bad gso: type: 1, size: 140
```

## System Information

version | core-2024.8.1
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.12.4
os_name | Linux
os_version | 6.6.44-haos
arch | x86_64
timezone | America/New_York
config_dir | /config

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
GitHub API Calls Remaining | 5000
Installed Version | 1.34.0
Stage | running
Available Repositories | 1392
Downloaded Repositories | 17

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | false
-- | --
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 13.0
-- | --
update_channel | stable
supervisor_version | supervisor-2024.08.0
agent_version | 1.6.0
docker_version | 26.1.4
disk_total | 30.8 GB
disk_used | 9.6 GB
healthy | true
supported | true
host_connectivity | true
supervisor_connectivity | true
ntp_synchronized | true
virtualization | kvm
board | ova
supervisor_api | ok
version_api | ok
installed_addons | File editor (5.8.0), Mosquitto broker (6.4.1), TasmoAdmin (0.30.4), ESPHome (2024.7.3), MQTT Explorer (browser-1.0.1), SQLite Web (4.2.0), Home Assistant Google Drive Backup (0.112.1), Advanced SSH & Web Terminal (18.0.0), Terminal & SSH (9.14.0)

</details>

<details><summary>Dashboards</summary>

dashboards | 5
-- | --
resources | 8
views | 12
mode | storage

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | July 20, 2024 at 9:21 PM
-- | --
current_recorder_run | August 14, 2024 at 6:52 PM
estimated_db_size | 1772.25 MiB
database_engine | sqlite
database_version | 3.45.3

</details>

**Comment by sairon** [comment]
This might be potential root cause of #3517 and #3528 and was fixed by #3529. We'll do a bit more of testing and release 13.1 with the fixed kernel ASAP.

**Comment by vampywiz17** [comment]
I also face these issue, but different simtome.. i can not able to upload anything that use TCP port... smb, scp, web etc...

it also affect my other server (Alpine linux) that run 6.6.44 same issue. 6.6.45 also bugged, 6,6,46 (edge versin in alpine) are working well again :)
